packages/lib-classifier/src/components/Classifier/components/MetaTools/MetaTools.js

```diff
@@ -42,6 +48,10 @@ export default class MetaTools extends React.Component {
             closeFn={this.toggleMetadataModal.bind(this)}
             metadata={subject.metadata}
           />}
+        <FavouritesButton
+          checked={this.state.isFavourite}
+          onClick={this.toggleFavourites.bind(this)}
```

Comment:
This should probably be bound in the constructor to avoid rebinding on subsequent renders

Reply:
Good point. Should be fixed now.